tests/lib/constants/constants.py

```diff
@@ -26,7 +26,9 @@
 
 TOKENS = [None, "Bearer xyzcba", "Bearer", "junk"]
 METHODS = ["get", "post", "put", "patch", "delete", "head", "options"]
+TEST_METHODS = ["get", "post", "put", "delete"]
```

Comment:
We should be using the existing set of `METHODS` for these tests I believe.

Reply:
I would only be inclined to agree if someone could give a reason as to why they are separate currently, or if we can agree as a consensus that we should migrate them to these constants.

Reply:
Ive tried running the tests with the additional methods but was met with failures for the options method